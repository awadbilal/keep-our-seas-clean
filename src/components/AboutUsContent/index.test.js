import React from "react";
import renderer from "react-test-renderer";
import AboutUsContent from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<AboutUsContent />).toJSON();
  expect(tree).toMatchSnapshot();
});
