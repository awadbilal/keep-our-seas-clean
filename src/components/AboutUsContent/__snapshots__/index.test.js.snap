// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="maincontent pt-5 pb-5 fdb-block container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-md-4 col-sm-12 offset-md-2"
      mt-5={true}
    >
      <h2
        className="title-about"
      >
        aboutUsContent.title
      </h2>
      <p
        className="aboutcontent  mt-4"
      >
        aboutUsContent.title.paragraph
      </p>
      <div
        className="row"
      >
        <div
          className="mt-5 col"
        >
          <img
            alt="alt"
            className="d-flex justify-content-between"
            src="vision.png"
          />
          <h5
            className="subTitle"
          >
            aboutUsContent.vission
          </h5>
          <p>
            aboutUsContent.vissionText
          </p>
        </div>
        <div
          className="mt-5 col"
        >
          <img
            alt="alt"
            className="d-flex justify-content-between"
            src="vision.png"
          />
          <h5
            className="subTitle"
          >
            aboutUsContent.mission
          </h5>
          <p>
            aboutUsContent.missionText
          </p>
        </div>
      </div>
    </div>
    <div
      className="col-md-6 col-sm-12"
    >
      <img
        alt="alt"
        className="groupImage"
        src="aboutimage.png"
      />
    </div>
  </div>
</div>
`;
